core/common/src/LocalDateTime.kt

```diff
@@ -77,7 +77,7 @@ import kotlinx.serialization.Serializable
  * LocalDateTime(date, time)
  * ```
  *
- * Some additional constructors that accept the date's and time's fields directly are provided for convenience.
+ * Some additional constructors that accept the date and time fields directly are provided for convenience.
```

Comment:
The idea here is that you can either pass the contents of the `date` and `time` fields (like above) *or* pass the fields *of* the date and the fields *of* the time. Hence the possessive form. Without it, I don't think the difference between the two approaches is highlighted. Maybe there's a better way to word this.

Reply:
To avoid confusion, I think it's better to write it explicitly:

"Some additional constructors that directly accept the values (or content) from date and time fields are provided for convenience."